docs/guidelines/platforms/android/guideline_understandable_android.md

```diff
@@ -107,6 +107,34 @@ In the given example, services such as TalkBack will read – "EditBox for usern
 
 ---
 
+### Accessible Authentication (Minimum) (WCAG 3.3.8 - Level AA)
+
+This guideline covers point [3.3.8 Accessible Authentication (Minimum) - Level AA](https://www.w3.org/WAI/WCAG22/quickref/#accessible-authentication-minimum) of the WCAG standard.
+
+To make authentication accessible to all users, the application should not require a cognitive function (such as remembering a password or solving a puzzle) test unless that step provides at least one of the following:
+
+- **Alternative**: Another authentication method that does not rely on a cognitive function test.
+- **Mechanism**: A mechanism is available to assist the user in completing the cognitive function test.
+- **Object Recognition**: The cognitive function test is to recognize objects.
+- **Personal Content**: The cognitive function test is to identify non-text content the user provided to the application.
+
+:white_check_mark: **Success criteria**
+
+- Support for text (copy and) paste to reduce the cognitive burden of re-typing
+- Support for password entry by password managers
```

Comment:
I know that this is not something that is written in the guidlines but I am sure it is because it is outdated. However, I would mention here passkey authentication. Something like this:

- Support for password entry by password managers or passkeys (i.e [Google passkey](https://developers.google.com/identity/passkeys))